test/replication/qsync_advanced.result

```diff
@@ -80,6 +80,9 @@ box.space.sync:insert{1} -- success
  | ---
  | - [1]
  | ...
+test_run:wait_lsn('replica', 'default')
```

Comment:
In this particular case `wait_lsn` isn't needed, because the sync insertion is done with quorum = 2, so the record must be present on replica already.